Backend/src/main/java/ktsnvt/tim1/model/Authority.java

```diff
@@ -0,0 +1,41 @@
+package ktsnvt.tim1.model;
+
+import javax.persistence.*;
+import java.util.HashSet;
+import java.util.Set;
+
+@Entity
+public class Authority {
+	@Id
+	@GeneratedValue(strategy= GenerationType.IDENTITY)
+	private Long id;
+	
+	String name;
```

Comment:
It would be better that name is UserType enumeration. Furthermore, an access modifier should be set to `private`.